src/Execution/ReportErrorHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution;
+
+use Closure;
+use GraphQL\Error\Error;
+use Illuminate\Contracts\Debug\ExceptionHandler;
+
+/**
+ * Report errors through the default exception handler configured in Laravel.
+ */
+class ReportErrorHandler implements ErrorHandler
+{
+    public static function handle(Error $error, Closure $next): array
+    {
+        $previous = $error->getPrevious();
+
+        // If the Error does not wrap another Error, it is related to a client error
+        // and shown in the error response anyway, we don't really need to report it
+        if ($previous) {
+            /** @var \Illuminate\Contracts\Debug\ExceptionHandler $reporter */
+            $reporter = app(ExceptionHandler::class);
```

Comment:
This could be moved to dependency injection through the `__constructor` right? 

The should work because the pipeline should resolve the error handlers from the container.

Reply:
Not right now, as the interface for the `ErrorHandler` requires the method to be `static`. I plan to change that in v5.

Reply:
Woops, missed the static keyword 🤦‍♂️, good that is planned on chaning in v5, might cleanup the handlers a bit 👍